pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -65,7 +65,7 @@ func (cmd Command) installCommand(c utils.CommandLine) error {
 // InstallPlugin downloads the plugin code as a zip file from the Grafana.com API
 // and then extracts the zip into the plugins directory.
 func InstallPlugin(pluginName, version string, c utils.CommandLine, client utils.ApiClient) error {
-	pluginFolder := c.PluginDirectory()
+	pluginsDir := setting.PluginsPath
```

Comment:
Where does `setting.PluginsPath` get set to the same value as the `--pluginsDir` option to grafana-cli?